### PR TITLE
Preview 2022 images

### PR DESCRIPTION
**Pull request by anamnavi**
## PR Summary

Create Preview images for nanoserver 2022 and windowsservercore 2022

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/preview/nanoserver2022/meta.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "preview-nanoserver-#shorttag#"
+    ],
+    "manifestLists": [
+        "nanoserver"
+    ],
+    "Base64EncodePackageUrl": true,
+    "TestProperties": {
+        "size": 1
+    },
+    "UseAcr": true,
```
Comment (RDIL) [inline release/preview/nanoserver2022/meta.json:22]: Does ACR support these newer images yet?

Comment (anamnavi) [reply release/preview/nanoserver2022/meta.json:22]: ACR has been updated to test 2022, but will be marked internal (via `isPrivate`)

Comment (TravisEz13) [reply release/preview/nanoserver2022/meta.json:22]: The public ACR doesn't support this yet... We have access to a VERY private prerelease that does.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**